app/src/main/java/com/mashup/mobalmobal/ui/profile/domain/model/ProfileItemMapper.kt

```diff
@@ -0,0 +1,72 @@
+package com.mashup.mobalmobal.ui.profile.domain.model
+
+import com.mashup.mobalmobal.R
+import com.mashup.mobalmobal.ui.profile.data.dto.MyDonateDto
+import com.mashup.mobalmobal.ui.profile.data.dto.MyPostDto
+
+fun MyDonateDto.toProfileItems(headerStringId: Int): List<ProfileItem> {
+    return mutableListOf<ProfileItem>().also {
+        if (donates.isNullOrEmpty()) return emptyList()
+
+        it.add(
+            ProfileItem.Header(
+                titleId = headerStringId
+            )
+        )
+
+        it.addAll(
+            donates.map { donate ->
+                ProfileItem.Donation(
+                    donationId = donate.postId.toString(),
+                    imageUrl = donate.post.postImage ?: "",
+                    title = donate.post.title,
+                    description = donate.post.description ?: "",
+                    goalPrice = donate.post.goalPrice.toDouble(),
+                    donatedPrice = donate.post.currentAmount.toDouble(),
+                    startAt = donate.post.startedAt ?: "",
+                    endAt = donate.post.endAt ?: ""
+                )
+            }
+        )
+    }
```

Comment:
```suggestion
        listOf(ProfileItem.Header(titleId = headerStringId) + donates.map { donate ->
                ProfileItem.Donation(
                    donationId = donate.postId.toString(),
                    imageUrl = donate.post.postImage ?: "",
                    title = donate.post.title,
                    description = donate.post.description ?: "",
                    goalPrice = donate.post.goalPrice.toDouble(),
                    donatedPrice = donate.post.currentAmount.toDouble(),
                    startAt = donate.post.startedAt ?: "",
                    endAt = donate.post.endAt ?: ""
                )
            }
    }
```
이렇게 사용하면 굳이 mutableList를 만들지 않아도 될 것 같아요

Reply:
좋은방법이네요!! 수정했습니다.